docs/orchestration/concepts/automations.md

```diff
@@ -0,0 +1,109 @@
+# Automations <Badge text="Cloud"/>
+
+Automations allow you to configure actions (such as cancelling a flow run or sending a notification to certain endpoints) when your flows or agents encounter certain events.
+
+For example, you can send a Slack message to your team when a run from a production critical flow has failed, along with the reason for the failure so that you can respond immediately. Or you can cancel a run that has been running (or scheduled) for more than an hour to enforce SLAs.  The Prefect Cloud API currently supports notification actions for Slack, Twilio, Pager Duty, email and arbitrary webhooks.
+
+You can set up and edit your Automations using the [API](/orchestration/concepts/api.html) or the Automations page in the UI.
+
+## Events
+
+Currently you can configure automations for the following events:
+- A flow run enters a given state or states
+- Runs from multiple (or all!) flows enter a given state or states
+
+Additionally, if you are a Standard Tier user, you can configure automations for the following scenarios:
+- A flow run fails to start after being scheduled for a certain amount of time
+- A flow run fails to finish after running for a certain amount of time
+- All agents with the same agent-config-id become unhealthy
+
+## Actions
```

Comment:
what is an `Action`?

Reply:
Note this also covers a small subset of available actions and we should probably add the rest

Reply:
Yes, we should expand this:
Action is a response on the [event](/orchestration/concepts/automations.md#events). You can send notifications to following services:
- Slack
- Twilio
- PagerDuty
- MS Teams
- Email
- Custom webhook
- Prefect UI
Moreover, you can 
- Cancel flow run

Reply:
The PrefectMessage / UI notification is not available publically rn.

Reply:
@madkinsz how would you prefer I document these additional hooks slash would you like to take over this branch?

Reply:
I can do it tomorrow

Reply:
@zhen0 -- I'm pretty swamped with Jim out. Could you take this back over?

Reply:
We could also add a note that documentation for other actions is coming soon and just merge this as is.

Reply:
Thanks for the updates. I'll take a look at this tomorrow.